src/WhatToCook.Application/Services/MealPlanningService.cs

```diff
@@ -47,21 +48,23 @@ public async Task<PlanOfMeals> Update(UpdatePlanOfMealRequest planOfMealRequest)
 
         if (mealPlanToUpdate == null)
         {
-            _logger.LogError($"Attempted to update a non-existent meal plan: {planOfMealRequest.Name}");
+            _logger.LogError("Attempted to update a non-existent meal plan{planOfMealName}", planOfMealRequest.Name);
```

Comment:
add whitespace between plan and {planOfMealName}

Reply:
will be in the next commit